src/views/ViewView.vue

```diff
@@ -89,6 +91,25 @@ export default defineComponent({
       updateLoadingStdErr,
       updateLoadingStdWin,
     } = useLoading();
+
+
+     // TODO: generalize logic more + page to allow multiple calls/groups
+     // based on different values ('open', 'closed', etc.)
+     function filterInOpenStatusNFT (arr: Array<INFT>) {
+       // Takes in an array of NFTs & filters to just "open" ones
+       // Assumes metatadataExternal attributes are fixed where index 1 {} is trait-type status
+       let arr_keep: Array<INFT> = []
+       for (var n of arr){
+          if (n.metadataExternal.attributes.length >= 2){
+              if (n.metadataExternal.attributes[1].value == 'open'){
```

Comment:
Not critical, but check out creating a lambda function inline that u can pass into the .filter() function so u dont have to do if checks and push (its also more performant): https://medium.com/lambda-js/meeting-lambda-3-filter-4bff5991311c